settings.gradle

```diff
@@ -17,6 +17,10 @@
 pluginManagement {
   repositories {
     gradlePluginPortal()
+    maven {
```

Comment:
Is this needed?  There are no plugins from the `tech.pegasys` namespace.

Reply:
it is needed because new web3j plugin has `tech.pegasys:jc-kzg-4844` as dependency

Reply:
That's not good.  Web3j cannot be loaded entirely from maven central?  Consensys should invest time in getting these published to Maven Central.

Reply:
I know someone is already working on that, so this will not be required in a future version, in the meantime what you prefer, using version `4.10.0` of the plugin or temporarily keeping the extra repo here?

Reply:
We do have the repository in our general listings.  I was hoping we could keep the plugins section minimal.  So this is acceptable since a long term solution is progressing.